android/autodispose-androidx-lifecycle/src/androidTest/java/autodispose2/androidx/lifecycle/AndroidLifecycleScopeProviderTest.java

```diff
@@ -103,11 +103,11 @@ public void observable_startStop() {
     final PublishSubject<Integer> subject = PublishSubject.create();
 
     // Spin it up
-    TestLifecycleOwner lifecycle = TestLifecycleOwner.create();
-    lifecycle.emit(Lifecycle.Event.ON_CREATE);
-    lifecycle.emit(Lifecycle.Event.ON_START);
+    TestLifecycleOwner lifecycle = new TestLifecycleOwner();
+    lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_CREATE);
+    lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_START);
```

Comment:
In this case, the default state of `TestLifecycleOwner` of `STARTED` means you can remove the `ON_CREATE` and `ON_START` calls here.

Reply:
nice, good catch